.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: npm-package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+
+  publish-npm:
```

Comment:
How would you handle change of version in tezster.js?

Reply:
That will auto fetch for tezster version from package.json file. And have to merge PR into master after changing versions in tezster.js and package.json both files.